src/components/dataset/summary/row.vue

```diff
@@ -116,6 +116,7 @@ export default {
 
     .property-list {
       hyphens: auto;
+      word-break: break-word;
```

Comment:
I'm seeing that `break-word` is deprecated. What do you think about using `overflow-wrap: break-word;`? We use that elsewhere, and it seems to do the right thing.